Code Style: Recommend initializing objects w properties (#21)
elementary/docs: writing-apps/code-style.md

can_focus = false;
```

### Initialize Objects with Properites

> This is especially more clear when initializing an object with many properties. Avoid the following

Reviewer: Should be "clearer" not "more clear"

> This is especially more clear when initializing an object with many properties. Avoid the following

Reviewer 2: ```suggestion
This is especially clearer when initializing an object with many properties. Avoid the following
```

> ```csharp

Reviewer 3: highlightjs actually supports vala so you might want to switch to ```vala

> ```csharp

Reviewer 2: ```suggestion
```vala
```

var label = new Gtk.Label ("Test Label");
label.set_ellipsize (Pango.EllipsizeMode.END);
label.set_valign (Gtk.Align.END);
label.set_width_chars (33);
label.set_xalign (0);
```

and instead do this

> ```csharp

Reviewer 2: ```suggestion
```vala
```
